tests/1-basic.spec.ts

```diff
@@ -0,0 +1,80 @@
+/*
+Basic configuration testing
+- Check if Validation can be initialized and correctly configured
+- No in-depth testing of Validation functionality
+*/
+
+import { test, expect } from '@playwright/test';
+import { Validation } from '../src/index'
+
+// Extend the window object to include the Validation class
+declare global {
+  interface Window {
+    Validation: typeof Validation;
+  }
+}
+
+test.describe('Form Validation Basic Tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('http://127.0.0.1:3000/tests');
+  });
```

Comment:
Does playwright start its own web server? I thought it just automated a browser.